serve.go

```diff
@@ -31,6 +33,8 @@ type config struct {
 	Key  string `env:"VHS_KEY" envDefault:""`
 	GID  int    `env:"VHS_GID" envDefault:"0"`
 	UID  int    `env:"VHS_UID" envDefault:"0"`
+
+	AuthorizedKeysPath string `env:"VHS_AUTHORIZED_KEYS_PATH" envDefault:"${HOME}/.ssh/authorized_keys" envExpand:"true"`
```

Comment:
Is this default going to be problematic for Windows users?

Reply:
good question... and I don't really know... what I did is to create the filepath if it does not exist, so at worst it will use an empty file... maybe we should change it to default empty though?

Reply:
Also, what if someone wants to keep a publicly accessible server?

Reply:
> Is this default going to be problematic for Windows users?

Assuming `${HOME}` is using the `os.Expand` functionality behind the scenes, technically Windows would use `USERPROFILE` and so this might not work as intended.
If possible, `os.UserHomeDir` would probably be preferable.

----

Perhaps to tackle both concerns with one stone, this could be optional entirely. If set, the user will be able to describe their home path as needed, and if not set then there is no auth.